terraform_files/vsphere/main.tf

```diff
@@ -0,0 +1,93 @@
+provider "vsphere" {
+  user                 = var.vsphere_username
+  password             = var.vsphere_password
+  vsphere_server       = var.vsphere_vcenter
+  allow_unverified_ssl = true
+}
+
+data "vsphere_datacenter" "datacenter" {
+  name = var.vsphere_datacenter
+}
+
+data "vsphere_compute_cluster" "cluster" {
+  name          = var.vsphere_cluster
+  datacenter_id = data.vsphere_datacenter.datacenter.id
+}
+
+data "vsphere_datastore" "datastore" {
+  name          = var.vsphere_datastore
+  datacenter_id = data.vsphere_datacenter.datacenter.id
+}
+
+data "vsphere_network" "network" {
+  name          = var.vsphere_network
+  datacenter_id = data.vsphere_datacenter.datacenter.id
+}
+
+resource "vsphere_tag_category" "category" {
+  name        = "openshift-${var.cluster_name}"
+  description = "Added by openshift-install do not remove"
```

Comment:
maybe expand description, or add comments to explain what's this value

Reply:
The description is the tag description which should be applied on every resource, i should be "Created by the test-infra, do not delete manually" but as you understand i just copied it from the Openshift installer and not ready for a review (just asked to publish the code to provide a picture of the changes)..... I will document everything properly